backend/app/views/spree/admin/orders/_risk_analysis.html.erb

```diff
@@ -1,50 +1,28 @@
-<fieldset class="no-border-bottom" id="risk_analysis">
-  <legend><%= "#{t('spree.risk_analysis')}: #{t('spree.not') unless @order.is_risky?} #{t('spree.risky')}" %></legend>
+<% risky_payments = @order.payments.risky.reorder("created_at DESC") %>
```

Comment:
Would you have any thoughts on pushing it to the controller? Queries on the view layer are always a red flag for me, and now is a good opportunity to move it.